intellijJVersions.gradle

```diff
@@ -101,9 +101,9 @@ static def ideProfiles() {
                     ]
                 ],
                 "RD": [
-                    sdkVersion  : "RD-2020.2-SNAPSHOT",
+                    sdkVersion  : "RD-2020.2",
```

Comment:
Can you upgrade the other ones too, running 2020.2 ultimate says "expired"

Reply:
Oh ya sure